libweasyl/alembic/versions/3ce027aa9bc4_track_oauth_consumer_app_homepages.py

```diff
@@ -0,0 +1,29 @@
+"""track oauth consumer app homepages
+
+Provide a way for OAuth2 consumer applications to
+register the homepage of their app, giving us
+a way to find out where our API's are being used
+as well as facilitating the possibility of featuring
+3rd party projects.
+
+Revision ID: 3ce027aa9bc4
+Revises: abac1922735d
+Create Date: 2016-05-09 16:02:48.478539
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '3ce027aa9bc4'
+down_revision = 'abac1922735d'
```

Comment:
The current revision on master is ``c8c088918278``, [here](https://github.com/Weasyl/weasyl/blob/master/libweasyl/libweasyl/alembic/versions/c8c088918278_index_media_by_sha256.py). Just wanted to give you a heads up since this might lead to a [branch situation](http://alembic.zzzcomputing.com/en/latest/branches.html), so it may be beneficial to update the ``down_revision``, unless we want to make a merge revision down the road to merge the branch this would make.

```
==> default: ve/bin/alembic upgrade head
==> default: INFO  [alembic.runtime.migration] Context impl PostgresqlImpl.
==> default: INFO  [alembic.runtime.migration] Will assume transactional DDL.
==> default: INFO  [alembic.runtime.migration] Running upgrade abac1922735d -> 9afc9a45510c, Add submission_tags for fast searching
==> default: INFO  [alembic.runtime.migration] Running upgrade 9afc9a45510c -> d0bffaa55b06, Index favorites by unixtime
==> default: INFO  [alembic.runtime.migration] Running upgrade d0bffaa55b06 -> 48ab4dc5599f, Do not index favorites by unixtime
==> default: INFO  [alembic.runtime.migration] Running upgrade 48ab4dc5599f -> c8c088918278, Index media by sha256
```